### ISSUE TITLE
Exception on first start of yast2 language module in Tumbleweed with xfce desktop

### ISSUE THREAD

**Issue by rhabacker** [yast/yast-country#235]
After installing Tumbleweed with xfce desktop I got an 'nil' exception on opening the yast2 language module in /usr/share/YaST2/modules/Keyboard.rb:306.

I managed the issuelocally by changing the mentioned file as shown below:
```
     # Returning current keyboard
     # @return [String] keyboard name e.g. "english-us"
     def current_kbd
-      self.Read if @curr_kbd.empty? && !Mode.config
+      self.Read if not @curr_kbd or @curr_kbd.empty? && !Mode.config
       @curr_kbd
     end
```

**Comment by dgdavid** [comment]
Hi @rhabacker!

Sorry, we overlooked your issue report. Actually, we don't use Github to track issues but https://bugzilla.opensuse.org/index.cgi instead. Among others, you can also reach us in the IRC `#yast` channel in the `freenode.net` server or through the yast-devel mailing list. See https://yast.opensuse.org/contact

The good news is that @jreidinger  already fixed the problem, although in a slightly different way :) See https://github.com/yast/yast-country/pull/238. The fix is available in yast2-country >= 4.2.14

Thank you so much for your collaboration!

**Comment by rhabacker** [comment]
Good to see that it has been fixed.
